config/i18n-tasks.yml

```diff
@@ -82,18 +82,20 @@ search:
 #   api_key: "AbC-dEf5"
 
 ## Do not consider these keys missing:
-# ignore_missing:
+ignore_missing:
+  - 'zxcvbn.*'
```

Comment:
It was finding the instance [here](https://github.com/18F/identity-idp/blob/1eca1c0efb6e694bdb4d73169d8912fb501f569d/app/assets/javascripts/misc/pw-strength.js#L32) and complaining since it wanted a value for `zxcvbn.feedback.`